protocol-units/da-sequencer/node/src/whitelist.rs

```diff
@@ -0,0 +1,77 @@
+use std::{
+	collections::HashSet,
+	fs,
+	path::PathBuf,
+	sync::{Arc, RwLock},
+	thread,
+	time::Duration,
+};
+
+use ed25519_dalek::VerifyingKey;
+use hex::FromHex;
+
+#[derive(Clone)]
+pub struct Whitelist {
+	pub(crate) inner: Arc<RwLock<HashSet<VerifyingKey>>>,
+}
+
+impl Whitelist {
+	fn load(path: &PathBuf) -> std::io::Result<HashSet<VerifyingKey>> {
+		let content = fs::read_to_string(path)?;
```

Comment:
Import the result above, then just `Result<HashSet<VerifyingKey>>`, reducing the chars in the fn sig.